src/dotnet/commands/dotnet-new/FSharp_Console/$projectName$.fsproj

```diff
@@ -1,5 +1,4 @@
-﻿<Project ToolsVersion="15.0">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+﻿<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
I see ok. Makes sense. For 3 - the FSharp.SDK;Microsoft.NET.Sdk is fine even longer term.